libs/api/domains/user-profile/src/lib/userProfile.resolver.ts

```diff
@@ -120,4 +127,12 @@ export class UserProfileResolver {
   ): Promise<DeleteTokenResponse> {
     return await this.userUserProfileService.deleteDeviceToken(input, user)
   }
+
+  @Mutation(() => ActorProfile)
+  async updateActorProfile(
```

Comment:
```suggestion
  @Mutation(() => ActorProfile, {name: 'updateUserProfileActorProfile'})
  async updateActorProfile(
```

or update the function name.